tests/insights/test_shap.py

```diff
@@ -101,20 +108,45 @@ def _test_shap_insight(campaign, explainer_cls, use_comp_rep, is_shap):
             use_comp_rep=use_comp_rep,
         )
 
-        # Sanity check explainer
+        # Sanity check explainers
         assert isinstance(shap_insight, insights.SHAPInsight)
-        assert isinstance(shap_insight.explainer, _get_explainer_cls(explainer_cls))
-        assert shap_insight.uses_shap_explainer == is_shap
+        # assert all(
+        #     isinstance(e, _get_explainer_cls(explainer_cls))
+        #     for e in shap_insight.explainers
+        # )
+        assert all(is_shap_explainer(e) == is_shap for e in shap_insight.explainers)
 
         # Sanity check explanation
         df = campaign.measurements[[p.name for p in campaign.parameters]]
         if use_comp_rep:
             df = campaign.searchspace.transform(df)
-        shap_explanation = shap_insight.explain(df)
-        assert isinstance(shap_explanation, shap.Explanation)
+        shap_explanations = shap_insight.explain(df)
+        assert isinstance(shap_explanations, tuple)
+        assert isinstance(shap_explanations[0], shap.Explanation)
```

Comment:
Why not simply check all of them?
```suggestion
        assert is_all_instance(shap_explanations, shap.Explanation)
```

Reply:
the test only has one but I guess this line looks nicer and doesnt hurt